css/menu.css

```
@@ -1,11 +1,16 @@
-body {
-  height: 100%;
-  padding-bottom: 80px;
-  padding-bottom: calc(80px + env(safe-area-inset-bottom));
+.no-supports-container .fl-menu-bottom-bar {
+  padding-bottom: 65px;
+  padding-bottom: calc(65px + env(safe-area-inset-bottom));
+}
+
+.supports-container .fl-page-content-wrapper {
+  padding-bottom: 65px;
+  padding-bottom: calc(65px + env(safe-area-inset-bottom));
 }
 
 [data-has-notch] body {
-  padding-bottom: 114px;
+  /* !import - to override the page.css style taht sometimes is added after the widget styles */
```

What does `!import` mean here? And I don't understand what this is here.

```suggestion
  /* !import - to override the page.css style that sometimes is added after the widget styles */
```

---

Typos fixed 😅